project/cmake/addons/README

```diff
@@ -44,7 +44,7 @@ specific paths:
   * ADDONS_DEFINITION_DIR points to the directory containing the definitions
     for the addons to be built.
   * ADDON_SRC_PREFIX can be used to override the addon repository location.
-    It must point to the locally available parent directory of the addon(s) to build
+    It should be a locally absolute parent directory of the addon(s) to build.
```

Comment:
It must point to the locally available parent directory of the addon(s) to build and must be an absolute path